code/stable/glassbr/SRS/JSON/GlassBR_SRS.ipynb

```diff
@@ -1360,7 +1360,7 @@
     "<li>$q&#770;_tol$ is the tolerable load (Unitless)</li>\n",
     "<li>$interpY$ is the interpY (Unitless)</li>\n",
     "<li>$AR$ is the aspect ratio (Unitless)</li>\n",
-    "<li>$J_tol$ is the stress distribution factor (Function) (Unitless)</li>\n",
+    "<li>$J_tol$ is the Tolerable stress distribution factor (Unitless)</li>\n",
```

Comment:
Should the capital be baked in? This may be a symptom of a larger issue (akin to #3534) that may be a separate spinoff issue (we love those!)

Reply:
Hahaha, I guess we should add this as a check to the smart constructors -- "error out if the user put in a capital letter". I'll fix it now!

Reply:
Ok, fixed that now too. Thank you @samm82 :smile:

Reply:
Another good catch @samm82.  I remember our discussion of capitalization.  I agree that the data in the Drasil DSLs should not have capitals.  We add those as needed.